source/manual/logging.html.md

```diff
@@ -89,6 +114,38 @@ Logstash is responsible for:
   Schema](https://www.elastic.co/guide/en/ecs/current/index.html)
 - loading the logs into Elasticsearch for storage and indexing
 
+### Vulnerability Scanners
```

Comment:
This section isn't really new, it's just moved from the bottom

Reply:
The section is introduced in a fairly awkward manner; it comes out of nowhere. I think it's fine for now.

Reply:
Hard agree on both counts :)